docs/en/glossary/glossary.asciidoc

```diff
@@ -551,6 +551,20 @@ nodes handling the user requests.
 endif::cloud-terms[]
 ifdef::elasticsearch-terms[]
 
+[[glossary-recovery]] recovery ::
+The process of updating a shard copy and making it available for queries.
```

Comment:
I think "updating" isn't super clear here. Neither was my "bring up to date". The idea is that recovery is about getting the on disk and in memory data structures "ready" and "current" based on data *somewhere*. On disk if this is the first node coming online with that shard copy. From the snapshot if you are restoring from a snapshot. Or from whatever node has the primary copy of the shard otherwise. I think that is all the cases, but this was never my area of expertise.

Reply:
Nor mine. 😄 

It seems obvious I'm gonna need to do some more digging here. Gonna throw a `WIP` label on this PR in the meantime.